### ISSUE TITLE
Does not compile on raspberry pi

### ISSUE THREAD

**Issue by vervaekejonathan** [Boscop/web-view#191]
This is the log, trying to compile for raspberry pi
Using the latest stable version and nightly, both give the same error.

```
   Compiling webview-sys v0.5.0
error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/github.com-1285ae84e5963aae/webview-sys-0.5.0/gtk.rs:132:13
    |
132 |             b"\0".as_ptr() as *const i8
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`
```

**Comment by lights0123** [comment]
This should be fixable by using `libc::c_char` instead.

**Comment by zxey** [comment]
@lights0123 proposed fix should work, if you have the time, send me a PR, I will merge it right away.

**Comment by lights0123** [comment]
@zxey wait this was actually fixed in #163, it just hasn't been version bumped + published

**Comment by vervaekejonathan** [comment]
oh my! 
tested the change on RPI, works smooth, thanks!

**Comment by zxey** [comment]
Published new version 0.6.3 to crates.io.
